assets/Script/Languages/cz.ts

```diff
@@ -115,66 +115,66 @@ export const CZ = {
     CarFactory: "Továrna na auta",
     Ship: "Loď",
     SolarPanel: "Solární panel",
-    SolarPanelDesc: "💡 Solární panely fungují jenom 5 vteřin každých 10 vteřin - všechny cykly jsou vyrovnané",
+    SolarPanelDesc: "💡 Solární panely fungují jenom 5 vteřin z každých 10 vteřin - všechny cykly jsou vyrovnané",
     WindTurbineDesc: "💡 Větrné turbíny fungují jenom 8 vteřin z každých 10 vteřin - cykly nejsou vyrovnané.",
     Movie: "Movie",
-    MovieStudio: "Movie Studio",
+    MovieStudio: "Filmové studio",
     Sitcom: "Sitcom",
-    TVStudio: "TV Studio",
+    TVStudio: "TV studio",
     PC: "PC",
```

Comment:
```suggestion
    PC: "Počítač",
```
...to align with "Továrna na počítače"